packages/inferno-animation/src/AnimatedComponent.ts

```diff
@@ -83,14 +83,14 @@ export default class AnimatedComponent extends Component<Props> {
     // Needs to be done after activating so timeout is calculated correctly
     registerTransitionListener(dom, function () {
```

Comment:
Can we directly set  registerTransitionListener(dom, callback, false) here to avoid creating function context?

Reply:
Will investigate

Reply:
@Havunen from what I see we have three alternatives:

1. current solution using closure
2. using .bind on a component method and creating an object that contains dom and animCls to pass as this
3. setting dom and animCls on the component and accessing it from method that is added to the component

Alt. 2 I don't think has any performance benefit
Alt. 3 pollutes the component and could cause unexpected bugs

Both alt. 2 & 3 reduce code readability quite a lot, so I am not sure if it is worth it given that animations are expensive operations anyway.

How do you feel about it?

Reply:
ok